src/js/core/config.local

```diff
@@ -0,0 +1,114 @@
+export default {
```

Comment:
Could you remove this file from the PR? It should get replaced soon anyways

Reply:
Removed